### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/quotes.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4865]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4865|
|Type|enhancement|
|Created|2011-02-11 05:15:10Z|
|Branch|https://github.com/twisted/twisted/tree/quotes-sphinx-4865|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the quotes howto

<details><summary>Searchable metadata</summary>

```
trac-id__4865 4865
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_quotes_sphinx_4865 branches/quotes-sphinx-4865
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297401310000000 1297401310000000
changetime__1300138011000000 1300138011000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30662]) Branching to 'quotes-sphinx-4865'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30663]) Merge the quotes parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs [#4865](https://github.com/twisted/twisted/issues/4865)
refs #4568

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_4"></a>|@tenth10th set owner to Screwtape|
|-|-|

The submitted change, which allows the "<code>/mystuff/TwistedQuotes</code>" tag to render correctly, is great.

But on viewing the rendered page, I noticed that there are mysterious semicolons after the second and third Python code blocks... They are present in the original Lore version of the docs, but they look like typos to me. Unless they have some purpose, we should probably remove those as well?

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn commented|
|-|-|

Since the Mysterious Semicolons are not really a conversion issue, opened a new ticket: [#4944](https://github.com/twisted/twisted/issues/4944)

If there are no other issues, this one can be merged.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @lvh|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_8"></a>|@lvh set status to `closed`|
|-|-|

(In [31060]) Better lore2sphinx output for quotes.xhtml

Author: khorn, Screwtape, jdb
Reviewer: tenth, lvh
Fixes: [#4865](https://github.com/twisted/twisted/issues/4865)

"Make Sphinx look gooder" -- JP
